channels/rail/rail_common.c

```diff
@@ -150,3 +151,190 @@ void rail_write_handshake_ex_order(wStream* s, const RAIL_HANDSHAKE_EX_ORDER* ha
 	Stream_Write_UINT32(s, handshakeEx->buildNumber); /* buildNumber (4 bytes) */
 	Stream_Write_UINT32(s, handshakeEx->railHandshakeFlags); /* railHandshakeFlags (4 bytes) */
 }
+
```

Comment:
those just moved to common? any changes?

Reply:
No changes